<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_tracker_alert</class>
 <widget class="QWidget" name="widget_tracker_alert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxAlertIfStayInRoiDuration">
     <property name="title">
      <string>Alert if same target stay in the roi</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelDuration">
        <property name="text">
         <string>Duration(sec)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxAlertIfStayInRoiDuration">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3600</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="horizontalSliderAlertIfStayInRoiDuration">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3600</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxSaveReports">
     <property name="text">
      <string>Save reports</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>317</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxAlertIfStayInRoiDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderAlertIfStayInRoiDuration</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlertIfStayInRoiDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxAlertIfStayInRoiDuration</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
